packages/playground/src/__tests__/api/annotationsApi.int.spec.ts

```diff
@@ -48,6 +48,10 @@ const FILE_FILTER: AnnotationFilterProps = {
   annotatedResourceIds: [{ externalId: ANNOTATED_FILE_EXTERNAL_ID }],
 };
 
+/**
+ * Annotations API tests are temporarily skipped due to the tests failing, which blocks non-related releases.
+ * The Annotation endpoint has changed behavior, leading these tests to produce 400 errors.
+ */
```

Comment:
nit: I'd say that this type of text should be a regular comment (`//` or `/*`) and not a "JSDoc" (or TSDoc or whatever we would call this) using `/**`. A JSDoc or similar would be appropriate if we were to _document_ an interface, symbol or something similar. This is internal (implementation) details to this file only describing _why_ the code is as it is.

```suggestion
// Annotations API tests are temporarily skipped due to the tests failing, which blocks non-related releases.
// The Annotation endpoint has changed behavior, leading these tests to produce 400 errors.
```